src/test/java/pdfverification/tests/HgvPRSBilingualTests.java

```diff
@@ -0,0 +1,57 @@
+package pdfverification.tests;
+
+import com.github.jknack.handlebars.Handlebars;
+import com.itextpdf.text.pdf.PdfReader;
+import htmlverification.service.CvsCertificateTestDataProvider;
+import org.junit.Before;
+import org.junit.Test;
+import org.xhtmlrenderer.pdf.ITextRenderer;
+import pdfverification.service.PDFParser;
+import uk.gov.dvsa.model.cvs.CvsMotCertificate;
+import uk.gov.dvsa.service.HtmlGenerator;
+import uk.gov.dvsa.service.PDFGenerationService;
+
+import java.io.IOException;
+
+import static org.junit.Assert.assertTrue;
+
+public class HgvPRSBilingualTests {
+    private static final String CERT_NAME = "MOT test certificate (HGV)";
+    private static final String CERT_NAME_FAIL = "Refusal of MOT test certificate";
+    private static final String CERT_NAME_WELSH = "Tystysgrif prawf MOT (HGV)";
+    private static final String CERT_NAME_FAIL_WELSH = "Gwrthod tystysgrif prawf MOT";
+
+    private HtmlGenerator htmlGenerator;
+    private PDFGenerationService pdfGenerationService;
+    private CvsMotCertificate testCertificate;
+    private PDFParser pdfParser;
+    private PdfReader pdfReader;
+    private byte[] pdfData;
```

Comment:
```suggestion
     private final HtmlGenerator htmlGenerator;
    private final PDFGenerationService pdfGenerationService;
    private CvsMotCertificate testCertificate;
    private final PDFParser pdfParser;
    private PdfReader pdfReader;
```

Reply:
Not sure what this suggested change is??

Reply:
Remove the variable and use locally?